tests/ExpirationValueTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: thomaskolonko
+ * Date: 15.10.18
+ * Time: 13:13
+ */
+
+use PHPUnit\Framework\TestCase;
+
+class ExpirationValueTest extends TestCase
+{
+    public function testYolo() {
```

Comment:
Please remove and add some proper testing. Most of your logic is DB logic. However, there might be the one or the other little thing you can test here. e.g. getThresholdBoundary. Also hardly think on how you can increase the test coverage here.